docs/AppFramework.md

```diff
@@ -17,8 +17,12 @@ Utilizing the App Framework requires:
 In this example, you'll deploy a Standalone CR with a remote storage volume, the location of the app archives, and set the installation location for the Splunk Enterprise Pod instance by using `scope`.
 
 1. Confirm your S3-based remote storage volume path and URL.
-2. Create a Kubernetes Secret Object with the storage credentials. 
-   * Example: `kubectl create secret generic s3-secret --from-literal=s3_access_key=AKIAIOSFODNN7EXAMPLE --from-literal=s3_secret_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`
+
+2. Configure credentials to connect to remote store by either:
+   * Configure roles for the Operator and Splunk instance pods using a service role or annotations, or
```

Comment:
I see 'roles' and 'service role', but I don't get the diff between them.

Reply:
Ya this is confusing.  I mean to say this:
```
Configure an IAM role for the Operator and Splunk instance pods using a service account or annotations, or
```